index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section id="main">
+        <h1 id="title">The Pyramids</h1>
+        <p class="cap">Impressive ancient structure</p>
+        <div id="img-div">
+            <img src="./images/pyramids-2371501-1920.jpg" id="image">
+            <caption id="img-caption">The pyramids are more than 7000 years old</caption>
+        </div>
+        <div id="tribute-info">
+            <h2 style="text-align: center;">Description</h2>
```

Comment:
Inline CSS should not be used in index.html
please remove it and add it to styles.css